collect_app/src/main/java/org/odk/collect/android/geo/OsmDroidMapFragment.java

```diff
@@ -381,10 +381,11 @@ public void removeFeature(int featureId) {
 
     @Override
     public void clearFeatures() {
-        map.getOverlays().clear();
```

Comment:
I don't know why it was implemented in this way... clearing everything not just features like it is in google and mapbox and like I implemented it now.